### ISSUE TITLE
very slooooow

### ISSUE THREAD

**Issue by denmau-zz** [kelseyhightower/nocode#4342]
Took very long to open on my device

**Comment by 0o10** [comment]
Yes it seems they are still adressing an issue where you need a device https://github.com/kelseyhightower/nocode/issues/2

Hopefully this will be fixed soon.

However, in the meantime, I suggest trying to run this on two slices of American cheese. Before consumption it appeared to run much faster on those.

**Comment by Hocuri** [comment]
Why only two slices of cheese? Four slices make the code look way more clear. Also, everyone else uses four slices (or taps), and it is annoying to have to read two-slices-code occasionally.

**Comment by sfxworks** [comment]
It should be mentioned that this has successfully run on a WHOPPER (with pickles) per https://github.com/kelseyhightower/nocode/issues/2#issuecomment-363685487
